test/service/test_repository_proxy.py

```diff
@@ -100,23 +100,24 @@ def assertUrlEqual(self, a: furl, b: furl):
             new=MagicMock())
 @mock.patch('azul.service.source_cache_service.SourceCacheService.get')
 class TestTDRRepositoryProxy(RepositoryPluginTestCase):
+    mock_service_url = f'https://serpentine.datarepo-dev.broadinstitute.net.test.{config.domain_name}'
+    mock_source_names = ['mock_snapshot_1', 'mock_snapshot_2']
+    make_mock_source_spec = 'tdr:mock:snapshot/{}:'.format
+    mock_sources = set(map(make_mock_source_spec, mock_source_names))
+
     catalog = 'testtdr'
     catalog_config = {
         catalog: config.Catalog(name=catalog,
                                 atlas='hca',
                                 internal=False,
                                 plugins=dict(metadata=config.Catalog.Plugin(name='hca'),
-                                             repository=config.Catalog.Plugin(name='tdr')))
+                                             repository=config.Catalog.Plugin(name='tdr')),
+                                sources=mock_sources)
     }
 
-    mock_service_url = f'https://serpentine.datarepo-dev.broadinstitute.net.test.{config.domain_name}'
-    mock_source_names = ['mock_snapshot_1', 'mock_snapshot_2']
-    make_mock_source_spec = 'tdr:mock:snapshot/{}:'.format
-    mock_source_specs = ','.join(map(make_mock_source_spec, mock_source_names))
-
     @mock.patch.dict(os.environ,
```

Comment:
That's not how we wrap aligned indents:

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14line-wrapping-and-indentation

Anyways, no need to wrap.